### DISCUSSION TITLE
Is there a custom wallet connector boilerplate?

### DISCUSSION THREAD

**Discussion by chimalsky** [wevm/wagmi#2173] [Q&A]
I couldn't find a custom wallet connector boilerplate. It seems common enough that there might be one/some. I wanted to ask before I began creating my custom wallet connector from scratch.

Otherwise my approach will be referring to https://eips.ethereum.org/EIPS/eip-1193 as well as looking at already existing connectors such as Coinbase's.

Thanks for the superb library!

**Comment by tmm** [comment]
Nope, just need to extend the [base `Connector`](https://github.com/wagmi-dev/references/blob/main/packages/connectors/src/base.ts).

**Reply by percyzt** [reply]
How can I customize the connector for the OKX Wallet wallet?
